### DISCUSSION TITLE
T-TWR Plus Rev 2.0 Support

### DISCUSSION THREAD

**Discussion by TechMindsYT** [erstec/APRS-ESP#46] [General]
Hi there, I wonder if this works with the above mentioned device. I have used PlatformIO to upload the image, but there is no screen display and no Wifi AP available. Only a green LED illuminates nears the PTT button.

Any suggestions please?

Thanks
Matt

**Comment by erstec** [comment]
No, it not and will not till me @erstec or @lyusupov or someone else will implement ESP32S3 support.

**Reply by TechMindsYT** [reply]
Ok thanks, Release 1.8 states that T-TWR support was added, so kinda confusing as it's not specific on board version which is supported.
